### PR TITLE
Remove UnsupportedOsPlatform from CryptoConfig as a utility class

### PR DESCRIPTION
**Pull request by lewing**
Add back CryptoConfig as supported on browser.

### CODE REVIEW

File: src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CryptoConfig.cs [hunk 1]
```diff
@@ -12,7 +12,6 @@
 
 namespace System.Security.Cryptography
 {
-    [UnsupportedOSPlatform("browser")]
```
Comment (buyaa-n) [inline src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CryptoConfig.cs:15]: It also needs to be removed from ref, and please remove unused import `using System.Runtime.Versioning;`

Comment (lewing) [reply src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CryptoConfig.cs:15]: Thanks!

**Review by jeffhandley — APPROVED** [review]

**Review by akoeplinger — APPROVED** [review]

**Review by buyaa-n — APPROVED** [review]
Thanks!

**Comment by lewing** [comment]
Failures are unrelated to browser only change.

**Pull request merged**